pkg/config/config.go

```diff
@@ -129,6 +129,7 @@ type HarvesterChartValues struct {
 
 type Install struct {
 	Automatic           bool    `json:"automatic,omitempty"`
+	SkipChecks          bool    `json:"skipchecks,omitempty"`
```

Comment:
> IIRC, the default bool variable will be false. This means if we do not set `harvester.install.skipchecks=true` it will run check by default.

Correct.

> I know we introduced this parameter, but some older install scripts (maybe an ansible yaml file) would not notice that change. So, they will face some trouble if their hardware does not fit our expectations.

Yes, older install scripts would need to be updated to add this parameter.

> Could we default skip check instead? (pxe-only) This means only automatic=true -> then the default check is false. That will not break the older automation deployment. WDYT?

A couple of thoughts:

1. If we default to skipping for PXE, my concern is that nobody will actually bother to enable the checks :-) as PXE users would have to explicitly set `harvester.install.skipchecks=false` if they wanted that behaviour. If I'm right that few PXE users will turn the checks on, we could just get rid of the flag altogether, and always skip for PXE (with failed checks still logged to `/var/log/console.log`).
2. Alternately, we could keep the flag, set it to default skip for automatic installs for the 1.3.0 release, then later (say, the 1.4.0 release?), change the default back to false. This means we can ensure it's well documented in 1.3.0 that these new checks exist, and that you'll need to prepare any deployment scripts for a behaviour change in the _next_ release (1.4.0).

I don't really have a strong preference either way. WDYT?

Reply:
The concern makes sense. We need to make sure the change is well-mentioned in the release note. Record in the issue https://github.com/harvester/harvester/issues/1154#issuecomment-1909194169

Reply:
Thanks for the explanation. That makes sense to me! Let's keep the current design.